tools/codegen/dest/lazy_ir.py

```diff
@@ -254,8 +260,8 @@ def this_shape(i: int) -> str:
         auto result = torch::lazy::TupleAtenFromLtcTensors<{returns_length}>(lazy_tensors);"""
 
         if schema.name.name.inplace or func.func.is_out_fn():
-            assert returns_length == 1, "We assumed there was no such case where an op is an in-place variant " \
-                                        "and has tuple outputs."
+            assert returns_length <= 1, "We assumed there was no such case where an op is an in-place variant " \
```

Comment:
what triggered this change? We didn't support any other ops with `out` parameters before?

Reply:
ohh good catch. i might not need to land this now.  I think it was triggered when I tried to codegen the amp_foreach_... ops which are inplace ops.  I am not sure if I will end up using codegen for those ops in the end or just hand-implementing them.  They are non-functional ops anyway, which is bad, but I think they are safe to use since they happen at the last stage (and torch-XLA has been supporting them).  So, yea maybe i'll back this change out now and only add it later if needed.